src/components/Kv/KvControlledCheckbox.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+	<div
+		class="kv-controlled-checkbox"
+		@click="handleInput"
+		@keypress="handleKeyPressContainer"
+		:tabindex="tabindex"
+		role="checkbox"
+		:aria-checked="checked"
+	>
+		<label class="kv-controlled-checkbox-label">
+			<input
+				type="checkbox"
+				:checked="checked"
+				aria-hidden="true"
+				role="presentation"
+				tabindex="-1"
+			>
+			<slot v-if="hasDefaultSlot"></slot>
+			<span v-else-if="label">{{ label }}</span>
```

Comment:
Couldn't you just put the <span> within the <slot> for this? Then the span would be replaced if slot content was present.

Reply:
Can you clarify please?

Reply:
https://vuejs.org/v2/guide/components-slots.html#Fallback-Content

Reply:
Oh cool great call!

Reply:
Updated with fallback content for slot